install.sh

```diff
@@ -16,31 +16,60 @@ CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
 CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 CURL="${CURL:-curl}"
 
-EARLY_PACKAGES="gcc10 llvm brotli c_ares libcyrussasl libiconv libidn2 \
-libmetalink libnghttp2 libpsl libssh2 libtirpc libunistring openldap \
-rtmpdump zstd ncurses ca_certificates ruby libffi openssl nettle krb5 \
-p11kit libtasn1 gnutls curl git icu4c libedit"
+EARLY_PACKAGES=(gcc10 libiconv libssh2 ncurses ruby openssl git)
 
 ARCH="$(uname -m)"
+BREW=🍺
+
+function puts() {
+  reset='\e[0m'
+  
+  case ${2} in
+  lightred)
+    colorize='\e[1;31m';;
+  lightgreen)
+    colorize='\e[1;32m';;
+  yellow)
+    colorize='\e[1;33m';;
+  orange)
+    colorize='\e[0;33m';;
+  lightblue)
+    colorize='\e[1;34m';;
+  cyan)
+    colorize='\e[0;36m';;
+  *)
+    colorize=${2};;
+  esac
+  
+  case ${3} in
+  cover)
+    colorize=${colorize}'\e[1A\e[K';;
+  covercurrent)
+    colorize=${colorize}'\r';;
+  printf);;
+  *)
+    reset=${reset}'\n';;
+  esac
+  
+  printf "${colorize}${1}${reset}"
+}
 
-GREEN='\e[1;32m';
-RED='\e[1;31m';
-BLUE='\e[1;34m';
-YELLOW='\e[1;33m';
-RESET='\e[0m'
-
-if [ "${EUID}" == "0" ]; then
-  echo -e "${RED}Chromebrew should not be installed or run as root.${RESET}"
-  exit 1;
-fi
+function rootchecker() {
+  if [ ${EUID} == 0 ]; then
+    puts 'Chromebrew should not be installed or run as root.' yellow
+    puts "Trying to switch to user chronos..." yellow
+    su chronos - <<< "$(curl -L# ${URL}/install.sh)" ||\
```

Comment:
Something doesn't feel right about this.  If you do this, don't you want to append ` && exit` afterwards?  Otherwise, the install continues after this and basically installs twice?